content/docs/command-reference/exp/init.md

```diff
@@ -0,0 +1,137 @@
+systems and require certain software packages to be installed.
+
+Wrap the command with double quotes `"` if there are special characters in it
+like `|` (pipe) or `<`, `>` (redirection), otherwise they would apply to
+`dvc run` itself. Use single quotes `'` instead if there are environment
+variables in it that should be evaluated dynamically. Examples:
+
+```dvc
+$ dvc exp init "./a_script.sh > /dev/null 2>&1"
+$ dvc exp init './another_script.sh $MYENVVAR'
+```
+
+## Options
+
+- `-i`, `--interactive` - prompts user for the command to execute and different
+  paths for tracking outputs and dependencies, unless they are provided through
+  cli arguments explicitly. Interactive mode allows users to set those location
+  from default values or omit them.
+
+- `--explicit` - `dvc exp init` assumes default location of your outputs and
+  dependencies (which can be overriden from the config). By using `--explicit`,
+  it will not use those default values while initializing experiments. In
+  `--interactive` mode, prompt won't set default value and all the values for
+  the prompt needs to be explicitly provided, or omitted.
+
+- `--code` - override the a path to your source file or directory which your
+  experiment depends on. The default is `src` directory for your code.
+
+- `--data` - override the path to your data file or directory to track, which
+  your experiment depends on. The default is `data` directory.
+
+- `--model` - override the path to your models file or directory to track, which
+  your experiment depends on. `dvc exp init` assumes `models` directory by
+  default.
+
+- `--params` - override the path to
+  [parameter dependencies](/doc/command-reference/params) which your experiment
+  depends on. The default parameters file name is `params.yaml`. Note that
+  `dvc exp init` may fail if the parameters file does not exist at the time of
+  the invocation.
+
+- `--metrics` - override the path to metrics file to track, which your
+  experiment produces. Default is `metrics.json` file.
+
+- `--plots` - override the path to plots file or directory, which your
+  experiment produces. The default is `plots`.
+
+- `--live` - override the directory `path` for
+  [DVCLive](/doc/dvclive/user-guide/dvclive-with-dvc), which your experiment
+  will write logs to. The default is `dvclive` directory, which only comes to
+  effect when used with `--type=live`.
+
+- `--type` - selects the type of the stage to create. Currently it provides two
+  different kinds of stages: `default` and `live`. If unspecified, `default`
+  stage is created.
+
+  `default` stage creates a stage with `metrics` and `plots` tracked by DVC
+  itself, and does not track live-created artifacts (unless explicitly
+  specified).
+
+  With `--type=live`, `dvc exp init` will create a stage where `metrics` and
+  `plots` are tracked by `dvclive` itself (unless `--metrics` and `--plots` are
+  explicitly provided).
```

Comment:
It might be helpful to discuss the intended use cases. The default stage is for classical ML experiments, while live is for deep learning. We should probably also mention the checkpoints in live.

Reply:
Not sure though. That should be explained in DVCLive docs and possibly also in a future `exp init` guide? I'd just link to DVC here too.

Reply:
It shouldn't really be about dvclive at all. It's about deep learning scenarios for which checkpoints and dvclive were built.

Reply:
We can still put it in the guide. Just want to clarify this doesn't really belong in dvclive docs.